Assets/Scripts/UI/StatBar.cs

```diff
@@ -66,6 +66,14 @@ void Awake()
 
     private void UpdateMeter()
     {
+        // Avoid one peculiar bug where the RectTransform is null
+        // and the MissingReferenceException spam prevents the next scene from loading
+        if (baseRect == null)
```

Comment:
Seems very weird, is there something underlying here or is this just Unity UI jank (tm)?

Reply:
I suspect there's an underlying issue, though this is outside the scope of this PR so I just slammed inn a temporary fix :upside_down_face:

Reply:
This isssue is tied to #70 , so we'll keep toss this on there.